public/static/docs/command-reference/checkout.md

```diff
@@ -25,7 +25,9 @@ workspace data with the versions specified in the current DVC-files.
 that changes the DVC-files in the workspace.
 
 💡 For convenience, a Git hook is available to automate running `dvc checkout`
-after `git checkout`. Use `dvc install` to install it.
+after `git checkout`. See the
+[**Automating** example](#example-automating-dvc-checkout) below or
```

Comment:
do we need to put into `**`? looks excessive a bit - emoji, link, bold

Reply:
Haha no we don't. Idk why I did that. Bolding removed.